features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/mbed_lib.json

```diff
@@ -11,14 +11,17 @@
         "6lowpan-nd-psk-key": "{0xa0, 0xa1, 0xa2, 0xa3, 0xa4, 0xa5, 0xa6, 0xa7, 0xa8, 0xa9, 0xaa, 0xab, 0xac, 0xad, 0xae, 0xaf}",
         "6lowpan-nd-sec-level": 5,
         "6lowpan-nd-device-type": "NET_6LOWPAN_ROUTER",
-        "thread-pskd": "\"Secret password\"",
+        "thread-pskd": "\"abcdefghijklmno\"",
```

Comment:
PSKd is only used when commissioning as authentication key in dtls session so it is not thread network configuration parameter.

But other values that are in thread configuration set do require it mainly all other parameters below